### PR TITLE
Improve swan lake plugin gradle build 

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
$subject. 
Resolves https://github.com/ballerina-platform/plugin-intellij/issues/58

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by ThemiraChathumina — APPROVED** [review]
LGTM

File: build-config/checkstyle/build/checkstyle.xml
Comment (ThisaruGuruge) [inline build-config/checkstyle/build/checkstyle.xml:1]: Do we need to commit this file? In the checkstyle plugin, it always downloads the file. We can either remove this file or use this without downloading it every time.

File: build.gradle [hunk 1]
```diff
@@ -16,37 +16,39 @@
...
 }
 
-version = ballerinaPluginVersion
-
-repositories {
+ repositories {
+    gradlePluginPortal()
     mavenCentral()
     maven { url 'https://jitpack.io' }
 }
```
Comment (ThisaruGuruge) [inline build.gradle:29]: Better to manage these in the `settings.gradle` file.

File: build-config/checkstyle/build/checkstyle.xml
Comment (NipunaRanasinghe) [reply build-config/checkstyle/build/checkstyle.xml:1]: @ThisaruGuruge you are correct. This is commited mistakenly. We have removed the file with https://github.com/ballerina-platform/plugin-intellij/pull/62

File: build.gradle (hunk 1 above)
Comment (NipunaRanasinghe) [reply build.gradle:29]: @ThisaruGuruge Actually thats what I tried first, but it ended up with different errors while resolving dependencies from the mention repos. So this was temporarily added until we figure out the root cause. So I'll remove these entries with the fix

**Pull request merged**